catalog/app/components/Layout/Layout.tsx

```diff
@@ -49,11 +79,11 @@ export function Layout({ bare = false, dark = false, children, pre }: LayoutProp
   return (
     <Root dark={dark}>
       <NavBar.Provider>
-        {bare ? <NavBar.Container /> : <NavBar.NavBar />}
+        <ErrorBoundary>{bare ? <NavBar.Container /> : <NavBar.NavBar />}</ErrorBoundary>
         {!!pre && pre}
         {!!children && <M.Box p={4}>{children}</M.Box>}
         <M.Box flexGrow={1} />
-        {!!isHomepage && isHomepage.isExact && <Footer />}
+        <ErrorBoundary>{!!isHomepage && isHomepage.isExact && <Footer />}</ErrorBoundary>
```

Comment:
probably it's better to only wrap the actual component in the boundary.
also, i think you can shorten the condition, make it `isHomepage?.isExact && ...`